contrib/inventory/gce_googleapiclient.py

```diff
@@ -189,7 +223,9 @@ def main(args):
         ARGS = docoptcfg(DOCOPT_USAGE, env_prefix=ENV_PREFIX)
 
     log.debug(ARGS)
-    if None in ARGS.values():
+    print(ARGS)
+    if not ARGS.values():
```

Comment:
i don't understand why you changed this. `ARGS.values()` will probably never be empty because we have a default value (`API_VERSION`).